### ISSUE TITLE
Support AES-GCM from new IETF proposal

### ISSUE THREAD

**Issue by iNPUTmice** [open-keychain/open-keychain#1929]
Hi,

there is a [IETF proposal](https://tools.ietf.org/html/draft-ford-openpgp-format-00) that among other things specifies the use of AES-GCM. I would like to see support for this in OpenKeychain. Would this be a lot of work?

**Comment by dschuermann** [comment]
No, this could be done because AES-GCM is already implemented in Bouncy Castle. Do you know if this is supported by GnuPG?

**Comment by iNPUTmice** [comment]
I don't know about GnuPGP but OpenPGP.js [1] has support for it. This actually led me to ask for this feature because AES-GCM is apparently very fast in modern browsers.

[1]: https://github.com/openpgpjs/openpgpjs
